controlcenter/static/controlcenter/css/all.css

```diff
@@ -43,7 +44,6 @@
   background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAABGCAAAAAAoI6GBAAAAOElEQVR4ATWNhQEAMAjDoP/fO3cJkwrmTU1dprmfX/TLwfefA6J2xWEfh4wJ1NJLLTtuNbuYEpUWQ5QZZ928F4cAAAAASUVORK5CYII="), url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAAAAADFHGIkAAAAQElEQVR4AXXIOQHAAACEMPzbxMEp6NafjMG7XYiXeImXeImXeImXeImXeImXeImX+BE/4kf8iB/xI37Ej/gRvwPQEA5XmMlaIAAAAABJRU5ErkJggg==");
 }
 
-.controlcenter__widget__subtitle,
 .controlcenter__widget__tab {
```

Comment:
Why did you dropped this? It is actually [exists](https://github.com/byashimov/django-controlcenter/blob/71dea21ab0b9f1693c923f3fa824c5436eb850eb/controlcenter/templates/controlcenter/dashboard.html#L31).

Reply:
Looks like it was added in css directly. Should be added to styl and regenerate css.